docs/reference/2-protocols/2-device-auth.md

```diff
@@ -3,20 +3,67 @@ id: device-auth
 title: Device Authentication
 ---
 
-To access Golioth Service from a device, the device need to be properly created and provisioned.
+To access Golioth Services from a device, the device needs to be properly created and provisioned.
 
-You can manage device and project credentials using the Web Console or through the CLI.
+You can manage device level and project credentials using the Web Console or through the CLI.
```

Comment:
> the CLI

Is this referring to the CoAP CLI? If so, should probably have a link here.

Reply:
also, will leave this to the devrel team to improve over time